apps/meteor/ee/app/livechat-enterprise/server/hooks/beforeNewInquiry.js

```diff
@@ -13,24 +13,38 @@ callbacks.add(
 			return extraData;
 		}
 
-		const sla = Promise.await(
-			OmnichannelServiceLevelAgreements.findOneByIdOrName(searchTerm, {
-				projection: { dueTimeInMinutes: 1 },
-			}),
-		);
-		if (!sla) {
-			throw new Meteor.Error('error-invalid-priority', 'Invalid sla', {
+		const slaP = OmnichannelServiceLevelAgreements.findOneByIdOrName(searchTerm, {
+			projection: { dueTimeInMinutes: 1 },
+		});
+		const priorityP = LivechatPriority.findOneByIdOrName(searchTerm, {
+			projection: { _id: 1, priorityWeight: 1 },
+		});
+		const [sla, priority] = Promise.await([slaP, priorityP]);
```

Comment:
Didn't know Promise.await behaved like `Promise.all` too 🤔

Reply:
I don't trust that. Let me change it to Promise.all()